metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "author": {
+        "name": "bakzkndd",
+        "id": "343103632453009408"
+    },
+    "service": "Ro the Bot",
+    "description": {
+        "en": "Ro the bot is the most diverse discord bot! Just imagine a function, and it probably has it."
+    },
+    "url": ["ro-the-bot.ga", "status.ro-the-bot.ga"],
+    "version": "1.0",
+    "logo": "https://cdn.discordapp.com/avatars/547090981279891486/10fc0b186f58583d466672ac14027cb2.png?size=2048",

```

Comment:
Please don't use Discord's CDN, use something like imgur or is-inside.me

Reply:
Changed it to an imgur